client/src/components/uis/MessageListItem.tsx

```diff
@@ -113,6 +115,13 @@ const StyledMyMessage = styled.View`
   border-radius: 3px;
 `;
 
+const styles = StyleSheet.create({
+  url: {
+    color: light.link,
```

Comment:
Why not `${({ theme }) => theme.link}` here?

Reply:
@hyochan 
I’ve made a few adjustment to StyleSheet.
Thanks for your suggestion!